daffodil-japi/src/main/scala/org/apache/daffodil/japi/infoset/Infoset.scala

```diff
@@ -119,238 +100,14 @@ abstract class InfosetInputter extends SInfosetInputter {
-   * @throws Exception if there was an error and Daffodil should stop parsing
-   */
-
-  @throws[Exception]
-  def startSimple(diSimple: DISimple): Unit
-
-  /**
-   * Called by Daffodil internals to signify the end of a simple element.
-   *
-   * @param diSimple the simple element that is ended. Various fields of
-   *                 DISimple can be accessed to determine things like the
-   *                 value, nil, name, namespace, etc.
-   * @throws Exception if there was an error and Daffodil should stop parsing
-   */
-  @throws[Exception]
-  def endSimple(diSimple: DISimple): Unit
-
-  /**
-   * Called by Daffodil internals to signify the beginning of a complex element.
-   *
-   * @param diComplex the complex element that is started. Various fields of
-   *                  DIComplex can be accessed to determine things like the
-   *                  nil, name, namespace, etc.
-   * @throws Exception if there was an error and Daffodil should stop parsing
-   */
-  @throws[Exception]
-  def startComplex(diComplex: DIComplex): Unit
-
-  /**
-   * Called by Daffodil internals to signify the end of a complex element.
-   *
-   * @param diComplex the complex element that is ended. Various fields of
-   *                  DIComplex can be accessed to determine things like the
-   *                  nil, name, namespace, etc.
-   * @throws Exception if there was an error and Daffodil should stop parsing
-   */
-  @throws[Exception]
-  def endComplex(diComplex: DIComplex): Unit
-
-  /**
-   * Called by Daffodil internals to signify the beginning of an array of elements.
-   *
-   * @param diArray the array that is started. Various fields of
-   *                DIArray can be accessed to determine things like the
-   *                name, namespace, etc.
-   * @throws Exception if there was an error and Daffodil should stop parsing
-   */
-  @throws[Exception]
-  def startArray(diArray: DIArray): Unit
-
-  /**
-   * Called by Daffodil internals to signify the end of an array of elements.
-   *
-   * @param diArray the array that is ended. Various fields of
-   *                DIArray can be accessed to determine things like the
-   *                name, namespace, etc.
-   * @throws Exception if there was an error and Daffodil should stop parsing
-   */
-  @throws[Exception]
-  def endArray(diArray: DIArray): Unit
-}
-
 /**
  * The below classes are all just proxies of the classes implemented in the
  * core daffodil code. The reason for this is purely for documentation. When
  * only generate javadoc on the daffodil-japi subproject. By having this proxy
  * classes, we can document these classes and have a small and clean javadoc.
- */
-
-/**
- * [[InfosetOutputter]] to build an infoset represented as a scala.xml.Node
  *
- * @param showFormatInfo add additional properties to each scala.xml.Node for debug purposes
- */
-class ScalaXMLInfosetOutputter(showFormatInfo: Boolean = false) extends InfosetOutputterProxy {
```

Comment:
Also all gone temporarily. The runtime1.infoset versions are just being used by JAPI tests for now.